data/na/swagger/1-0-5.yaml

```diff
@@ -2076,7 +2076,7 @@ definitions:
         type: string
         maxLength: 1
         description: 'How the transaction was initiated. N = Standard Entry, A = Apple Pay, G = Android Pay, P = Google Pay, S = EMV Swipe, V = Visa Checkout, Y = Unimag Card Swiper, T = EMV Chip Tap, E = EMV Chip Dip'
```

Comment:
Just curious did we add google pay description in one of our previous tickets ?

Reply:
I added Google Pay to the description when I was working on this ticket previously, a couple of weeks ago

Reply:
Ok thanks.